platform/broadcom/sonic-platform-modules-accton/as7816-64x/utils/accton_as7816_monitor.py

```diff
@@ -83,8 +85,8 @@ def manage_fans(self):
            4: [max_duty, 57000, sys.maxsize],
         }
   
-        thermal = ThermalUtil()
-        fan = FanUtil()
+        thermal = self.ThermalUtil()
+        fan = self.FanUtil()
```

Comment:
Is this correct? Or do you intend to do this:

```python
        self.thermal = ThermalUtil()
        self.fan = FanUtil()
```

Reply:
No, it's not correct.  
Thanks for reminding me.